entrypoint/secretsencrypt/encryption_test.go

```diff
@@ -0,0 +1,59 @@
+package secretsencrypt
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+
+	"github.com/rancher/distros-test-framework/pkg/assert"
+	"github.com/rancher/distros-test-framework/pkg/testcase"
+)
+
+var _ = Describe("Test:", func() {
+
+	It("Start Up with no issues", func() {
+		testcase.TestBuildCluster(GinkgoT())
+	})
+
+	It("Validate Nodes", func() {
+		testcase.TestNodeStatus(
+			assert.NodeAssertReadyStatus(),
+			nil,
+		)
+	})
+
+	It("Validate Pods", func() {
+		testcase.TestPodStatus(
+			assert.PodAssertRestart(),
+			assert.PodAssertReady(),
+			assert.PodAssertStatus(),
+		)
+	})
+
+	It("Validate Secrets Encryption", func() {
+		testcase.TestSecretsEncrypt()
```

Comment:
nit: Can we rename to `TestSecretsEncryption`?

Reply:
fixed